fixtures/ikan/stryder-sfb150.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Stryder SFB150",
+  "categories": ["Dimmer"],
```

Comment:
I'm torn.  Normally I don't like fixtures to declare themselves to be "dimmers" (see #2577), but this one is doubly weird in that it does seem to have a (separate but fixture-specific?) dimmer, so ... ?

Reply:
It doesn't seem that you can exclude the 'categories' attribute, or have an empty list. So, what should we do for basic lights like this?

Reply:
I personally use Other, for now, but I don't think there's any official position yet.  Either way you choose, there are many other fixtures that do it the same (and the other) way, so once there is a policy we can easily enough update all of them at once.